.aios-core/data/tech-presets/nextjs-react.md

```
@@ -0,0 +1,931 @@
+---
+
+## Changelog
+
+| Date       | Version | Changes                                              |
+| ---------- | ------- | ---------------------------------------------------- |
+| 2026-01-28 | 1.1.0   | Update to Next.js 16+, replace Middleware with Proxy |
+| 2025-01-27 | 1.0.0   | Initial version based on DEVELOPMENT_GUIDE.md        |
+
+---
+
+_AIOS Tech Preset - Synkra AIOS Framework_
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)